uint16 gpio_id
uint8 edge_change_type
---
int32 callback_id
bool has_succeeded